### PR TITLE
Add ActionId for etor-ti

### PR DESCRIPTION
**Pull request by jorg3lopez**
This PR adds the actionId as part of the http header of the REST transport, when specified in the org settings by adding `actionId: header.reportFile.actionId` under `headers`.

Test Steps:
1. Work in progress

## Changes
- *Include a comprehensive list of changes in this PR*
- *(For web UI changes) Include screenshots/video of changes*

### Testing
- [ ] Tested locally?
- [ ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [ ] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [ ] Added tests?

### Process
- [ ] Are there licensing issues with any new dependencies introduced?
- [ ] Includes a summary of what a code reviewer should test/verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

## Linked Issues
- Fixes #issue

### CODE REVIEW

**Review by halprin — COMMENTED** [review]
@jorg3lopez and @rajeev-flexion, I don't see any code that converts the `reportId` for this step in the pipeline to the first `reportId` of the pipeline (which is the `submissionId` that works in the history API).  At least that's the understanding I have after talking with other ReportStream engineers.  You can read more in the Notes section of [our story](https://github.com/CDCgov/trusted-intermediary/issues/790).  Maybe I'm wrong and this new `reportId` works just fine in the history API?

**Comment by jorg3lopez** [comment]
@halprin I'll look into it, most likely it will need to be the received actionId instead of the send actionId.

File: prime-router/src/main/kotlin/transport/RESTTransport.kt [hunk 1]
```diff
@@ -56,7 +57,7 @@ import kotlinx.serialization.json.Json
 import org.json.JSONObject
 import java.io.InputStream
 import java.security.KeyStore
-import java.util.Base64
+import java.util.*
```
Comment (basiliskus) [inline prime-router/src/main/kotlin/transport/RESTTransport.kt:60]: Better to use explicit imports

**Review by basiliskus — COMMENTED** [review]
Did you mean to delete the .idea folder from the repo?

File: prime-router/src/main/kotlin/transport/RESTTransport.kt (hunk 1 above)
Comment (jorg3lopez) [reply prime-router/src/main/kotlin/transport/RESTTransport.kt:60]: Good catch. I don't know how that got in there. I'll check my intelliJ settings.

**Comment by jorg3lopez** [comment]
> Did you mean to delete the .idea folder from the repo?

Hmm. Don't know why they have it. I'll leave it alone and put it back. I had to delete it as I couldn't build the application.

File: prime-router/src/main/kotlin/azure/CheckFunction.kt [hunk 2]
```diff
@@ -345,6 +345,7 @@ class CheckFunction : Logging {
         try {
             val theRESTTransport = RESTTransport()
             val reportId = UUID.randomUUID().toString()
+            val actionId = 12345L
```
Comment (oslynn) [inline prime-router/src/main/kotlin/azure/CheckFunction.kt:348]: Should actionId not be a hard coded?

Comment (rajeev-flexion) [reply prime-router/src/main/kotlin/azure/CheckFunction.kt:348]: we can have a randomly generated UUID.

Comment (jorg3lopez) [reply prime-router/src/main/kotlin/azure/CheckFunction.kt]: Added ACTION_ID as top level

**Comment by jorg3lopez** [comment]
@oslynn Just double checking if RS has a threat model in case I need to update it.

**Review by oslynn — APPROVED** [review]

**Pull request closed without merging**